tests/testthat/test-mock.r

```diff
@@ -116,9 +116,11 @@ test_that("can mock hidden functions", {
 })
 
 test_that("can mock if package is not loaded", {
-  if ("package:devtools" %in% search())
-    skip('devtools is loaded')
-  with_mock(`devtools::add_path` = identity, expect_identical(devtools::add_path, identity))
+  if ("package:boot" %in% search())
+    skip('boot is loaded')
+  if (!length(find.package("boot", quiet=TRUE)))
```

Comment:
I guess we can skip that test if `devtools` is not installed (the original case in #247). It will be tested with `R CMD check`, because it's in "Suggests".

The `disposables` package is already on CRAN, it can create one-off packages for testing.